Services/Init/classes/class.ilInitialisation.php

```diff
@@ -836,8 +836,8 @@ protected static function initAccessHandling()
 		self::initGlobal("rbacadmin", "ilRbacAdmin",
 			 "./Services/AccessControl/classes/class.ilRbacAdmin.php");
 		
-		self::initGlobal("ilAccess", "ilAccessHandler", 
-			 "./Services/AccessControl/classes/class.ilAccessHandler.php");
+		self::initGlobal("ilAccess", "ilAccess",
```

Comment:
As mentioned above, global $ilAccess is now a "ilAccess" with the interface ilAccessHandler